docs/docsite/rst/collections_tech_preview.rst

```diff
@@ -115,8 +115,43 @@ The ``ansible-doc`` command requires the fully qualified collection name (FQCN)
 plugins directory
 ------------------
 
- Add a 'per plugin type' specific subdirectory here, including ``module_utils`` which is usable not only by modules, but by any other plugin by using their FQCN. This is a way to distribute modules, lookups, filters, and so on, without having to import a role in every play.
+Add a 'per plugin type' specific subdirectory here, including ``module_utils`` which is usable not only by modules, but by any other plugin by using their Fully Qualified Collection Name (FQCN). This is a way to distribute modules, lookups, filters, and so on, without having to import a role in every play.
 
+module_utils
+~~~~~~~~~~~~
+
+When working with ``module_utils`` in a Collection, the python ``import`` statement will need to take into account the Fully Qualified Collection Name (FQDN) along with the ``ansible_collections`` convention. The resulting import will look like ``from ansible_collections.{namespace}.{collection}.plugins.module_utils.{util} import {something}``
+
+Example snippet of a module using both default Ansible ``module_utils`` and
+those provided by a collection. In this example the Collection is
+``ansible_security``, the namespace is ``community``, and the ``module_util`` in
+question is called ``qradar`` such that the Fully Qualified Collection Name
```

Comment:
You've spelled out Fully Qualified Collection Name and introduced the acronym more than once . . . you can use the acronym alone now, certainly for the rest of the section, probably for the rest of the page.

Reply:
@samccann we can even just have `example_collection` or whatever ... that was just something I happened to have open in vim at the time so I cribbed some code.